setup_wordpress.sh

```diff
@@ -1,6 +1,112 @@
-#!/bin/bash
-# Please do not remove this line. This command tells bash to stop executing on first error. 
-set -e
+#!/bin/bash -xe
```

Comment:
you create VM using command `vagrant up`.

There is antoher command `vagrant provision` that allows you to rerun your scripts without recreating VM. Could you try to make it work?